README.md

```diff
@@ -4,6 +4,15 @@
 
 ## Installation
 
+This documentation is built with Sphinx, a Python documentation generator. You will thus need to install a [Python runtime](https://www.python.org/downloads/) to build it. The version to install is specified in the `runtime.txt` file.
+
+In order to avoid conflicting dependencies with other projects on your local machine, it is recommended to install its dependencies in a virtual environment. To create a virtual environment, run:
+
+```
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
```

Comment:
Explain how to quit the virtual environment? Here or at the end of this section for exemple with something like:
```suggestion

And you can deactivate that venv when you are done running `openfisca-doc` with `deactivate`.
```

Reply:
Once more, I believe users are either familiar with `venv` or using this only for OpenFisca (my case, for example :wink:) and thus don't have other Python operations that could conflict with that specific venv. If we add that information without explaining what it means to “activate” or “deactivate” a venv it's just more confusing than anything.